java/Cherepnin_Oleh/src/com/zoolatech/lecture6/tasks/_3/UserService.java

```diff
@@ -0,0 +1,21 @@
+package com.zoolatech.lecture6.tasks._3;
+
+import com.zoolatech.lecture6.tasks._3.storage.impl.UserCacheImpl;
+import com.zoolatech.lecture6.tasks._3.storage.impl.UserTableImpl;
+
+public class UserService {
+    private UserRepository repository;
+
+    {
+        repository = new UserRepository(new UserCacheImpl(), new UserTableImpl());
```

Comment:
you don't need an initialization block here: you can assign a value directly to the variable or you can use a constructor